e2e/_suites/ingest-manager/services.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/elastic/e2e-testing/e2e"
+	log "github.com/sirupsen/logrus"
+)
+
+// ElasticAgentInstaller represents how to install an agent, depending of the box type
+type ElasticAgentInstaller struct {
+	artifactArch      string // architecture of the artifact
+	artifactExtension string // extension of the artifact
+	artifactName      string // name of the artifact
+	artifactOS        string // OS of the artifact
+	artifactVersion   string // version of the artifact
+	image             string // docker image
```

Comment:
The location in the file system, used to determine which image to use to spin up the service.